iothub/service/src/Common/Extensions/CommonExtensions.cs

```diff
@@ -244,6 +244,16 @@ public static void AppendKeyValuePairIfNotEmpty(this StringBuilder builder, stri
             }
         }
 
+        /// <summary>
+        /// Check if the value is null or empty.
+        /// </summary>
+        /// <param name="value">The value to check.</param>
+        /// <returns>Returns true if the value is null or empty, otherwise returns false.</returns>
+        public static bool IsNullOrWhiteSpace(this string value)
+        {
+            return string.IsNullOrWhiteSpace(value);
+        }
```

Comment:
Was there anything explained in the related PR on why this was removed? Was it simply unused?

Reply:
Here is the related [PR](https://github.com/Azure/azure-iot-sdk-csharp/pull/2801). It appears that we were replacing the custom method `IsNullOrWhiteSpace()` with `string.IsNullOrWhiteSpace()` in this PR, as a result, this public API didn't have any references in SDK any more, so removed.

Reply:
I see, so this is an extension method that in reality already exists on the `string` class. 

Since the functionality does exist on the `string` class anyway, I'd be ok with us removing this and calling out in our release notes that we removed this duplicated functionality and users should use the "native" API on the string class directly.
Thoughts?

Reply:
We're removing this public API but users shouldn't be broken by that since they can call `string.IsNullOrWhiteSpace(value)` directly anyway, right?

Reply:
Yes, that's right. I will still remove this API and call this out in release note then.